readme.md

```diff
@@ -6,4 +6,4 @@ UpLeveled Stylelint defaults for styling in CSS, SCSS, Sass, CSS Modules and CSS
 
 ## Setup
 
-Follow the setup instructions for the [UpLeveled ESLint Config](https://www.npmjs.com/package/@upleveled/eslint-config-upleveled)
+Follow the setup instructions for the [UpLeveled ESLint Config](https://www.npmjs.com/package/eslint-config-upleveled)
```

Comment:
link to the updated version of eslint-config-package instead of old version of deprecated version